rustls/src/crypto/ring/ticketer.rs

```diff
@@ -100,10 +106,23 @@ impl ProducesTickets for AeadTicketer {
                 ciphertext
             })
             .ok()
+            .map(|r| {
```

Comment:
Don't love the `map()` for this side-effecty thing. Maybe just stick `?` on the `ok()` and deal with this separately?